src/storage/schema.py

```diff
@@ -28,20 +28,57 @@
 class AnalysisEntry(Base):
     __tablename__ = 'analysis'
 
-    uid = mapped_column(UID, ForeignKey('file_object.uid', ondelete='CASCADE'), index=True)
-    plugin = mapped_column(VARCHAR(64), nullable=False)
-    plugin_version = mapped_column(VARCHAR(16), nullable=False)
-    system_version = mapped_column(VARCHAR)
-    analysis_date = mapped_column(Float, nullable=False)
-    summary = mapped_column(ARRAY(VARCHAR, dimensions=1), nullable=True)
-    tags = mapped_column(MutableDict.as_mutable(JSONB), nullable=True)
-    result = mapped_column(MutableDict.as_mutable(JSONB), nullable=True)
+    uid = mapped_column(
+        UID,
+        ForeignKey('file_object.uid', ondelete='CASCADE'),
+        index=True,
+        comment='The UID of the file object on which this analysis was performed.',
+    )
+    plugin = mapped_column(VARCHAR(64), nullable=False, comment='The name of the analysis plugin (e.g. "file_type").')
```

Comment:
As a personal style preference, I like to put every argument on its own regardless of the linelength.
Would you mind splitting the lines here too?